sdk/iot/core/src/az_iot_core.c

```diff
@@ -10,6 +10,71 @@
 
 #include <_az_cfg.h>
 
+AZ_NODISCARD az_result az_iot_get_status(uint32_t status_int, az_iot_status* status)
```

Comment:
ok, I have been trying to think of a better name for this...but I can't :(

Reply:
it could get `az_iot_get_status_from_int()`?

Reply:
or `..from_uint32()`. kinda like the span API's

Reply:
https://github.com/Azure/azure-sdk-for-c/blob/e789649ee5d56003078e52ac96d45710c6715fac/sdk/core/core/inc/az_span.h#L270

Reply:
updated